Linux-Application/DomesdayDuplicator/advancednamingdialog.cpp

```diff
@@ -28,6 +28,11 @@
 #include "advancednamingdialog.h"
 #include "ui_advancednamingdialog.h"
 
+QString mintSide1;
```

Comment:
You don't want these to be global variables - they should be private member variables of the AdvancedNamingDialog class (like `ui` already is).